pkg/server/api.go

```diff
@@ -136,7 +136,26 @@ func (sh *APIHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	// we know we're at 3.2 in code.. serve directly, parsing is expensive...
 	// we're doing it during an HTTP request, and most notably before we write the HTTP headers
 	var serveConf *runtime.RawExtension
-	if reqConfigVer.Equal(*semver.New("3.2.0")) {
+	if reqConfigVer.Equal(*semver.New("3.4.0")) {
+		converted34, err := ctrlcommon.ConvertRawExtIgnitionToV3_4(conf)
+		if err != nil {
+			w.Header().Set("Content-Length", "0")
+			w.WriteHeader(http.StatusInternalServerError)
+			glog.Errorf("couldn't convert config for req: %v, error: %v", cr, err)
+			return
+		}
+		serveConf = &converted34
+
+	} else if reqConfigVer.Equal(*semver.New("3.3.0")) {
```

Comment:
I think RHCOS has been on 3.3 for awhile, and technically we've been feeding it 3.2 up to now. I don't think this would change any actual functionality though right?

Reply:
I didn't know that and wasn't sure if things cared if they got a different version than they asked for. 

It shouldn't change any actual functionality, but we can totally keep feeding them `3.2` if you're worried. 

That probably turns into "If they ask for a 3.x version  > our default, give them our default" .

Reply:
It should be fine to send a newer config version.  Ignition uptranslates older configs internally anyway.